synapse/app/_daemonize.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*
+# Inspired by https://github.com/thesharp/daemonize/blob/master/daemonize.py
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import fcntl
+import os
+import sys
+import signal
+import atexit
+import traceback
+
+
+class Daemonize(object):
+
+    """
+    Daemonize object.
```

Comment:
I see you've copied this from the original Daemonize implementation, but I don't think it's very useful. *obviously* it's a Daemonize object, but what does it do? if you can't think of anything useful to say, don't say anything at all.